backends/p4tools/modules/testgen/targets/bmv2/test/small-step/p4_asserts_parser_test.cpp

```diff
@@ -23,17 +24,17 @@
 #include "lib/null.h"
 
 #include "backends/p4tools/modules/testgen/targets/bmv2/p4_refers_to_parser.h"
-#include "backends/p4tools/modules/testgen/test/gtest_utils.h"
 
 /// Variables are declared in "test/gtest/env.h" which is already included in reachability.cpp
 // FIXME: Make these Util::Path
 extern const char *sourcePath;
 extern const char *buildPath;
 
-namespace P4::Test {
+namespace P4::P4Tools::Test {
+
 using namespace P4::literals;
 
-class P4AssertsParserTest : public P4ToolsTest {};
+class P4AssertsParserTest : public ::testing::Test {};
```

Comment:
Is there a conflict on the `testing` namespace? Or why use the global marker `::`?

Reply:
I believe this is an artifact of other tests using `::testing::Test` and things getting copied as needed. 
Eg., https://github.com/p4lang/p4c/blob/main/test/gtest/bitrange.cpp#L37

I can clean these up to `testing::Test`, if desired.